gazelle/configure.go

```diff
@@ -223,6 +240,7 @@ func (*JS) KnownDirectives() []string {
 	return []string{
 		"js_extension",
 		"js_root",
+		"js_pages_root",
```

Comment:
wondering if this should be `js_next_pages_root` since this is specific for nextjs